networkit/cpp/graph/GraphTools.cpp

```diff
@@ -522,5 +522,41 @@ std::vector<node> topologicalSort(const Graph &G,
     return topSort.getResult();
 }
 
+bool isBipartite(const Graph &graph) {
+    if (graph.isDirected()) {
+        throw std::runtime_error("The graph is not an undirected graph!");
+    }
+    constexpr signed char uncolored{-1};
+    constexpr signed char black{};
+    constexpr signed char red{1};
+    std::vector<signed char> colors(graph.upperNodeIdBound(), uncolored);
+    bool isBipartite{true};
```

Comment:
Use the `=` operator for assigning these variables. I understand that `{}` is more modern but here is equivalent to `=` and we should keep consistency with the rest of the codebase.